### PR TITLE
Implement custom UI for the eligibility step.

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Requires: https://github.com/ResearchKit/ResearchKit/pull/845

### CODE REVIEW

File: BridgeAppSDK/SBAToggleFormStep.swift [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    public func numberOfRows(inSection section: Int) -> Int {
+        return self.formItems?.count ?? 0
+    }
+    
+    public func reuseIdentifierForRow(at indexPath: IndexPath) -> String {
+        return SBAToggleTableViewCell.reuseIdentifier
+    }
+    
+    let minCellHeight: CGFloat = 120
+    let maxCellHeight: CGFloat = 175
```
Comment (Erin-Mounts) [inline BridgeAppSDK/SBAToggleFormStep.swift]: Could these be defined as constraints in the storyboard instead of being hardcoded?

**Review by Erin-Mounts — APPROVED** [review]
Question about hardcoded size constraints, otherwise looks good, merge when ready

File: BridgeAppSDK/SBAToggleFormStep.swift (hunk 1 above)
Comment (syoung-smallwisdom) [reply BridgeAppSDK/SBAToggleFormStep.swift]: Doesn't use a storyboard but should be possible to use IBDesignable properties. I'll look into it.

**Comment by Erin-Mounts** [comment]
looks good

**Pull request merged**